Tools/CISupport/ews-build/steps.py

```diff
@@ -2888,10 +2888,10 @@ def send_email_for_new_build_failure(self):
                 logs = logs.replace('&', '&amp;').replace('<', '&lt;').replace('>', '&gt;')
                 email_text += '\n\nError lines:\n\n<code>{}</code>'.format(logs)
             email_text += '\n\nTo unsubscribe from these notifications or to provide any feedback please email aakash_jain@apple.com'
-            self._addToLog('stdio', 'Sending email notification to {}'.format(change_author))
+            self._addToLog('stdio', f'Sending email notification to {change_author}')
```

Comment:
Unfortunately, we really _do_ need the `yield` mess here.

If we don't respect that this call is deferred, we're likely to get two different `stdio` logs if we raise an exception in `send_email_to_patch_author`

Reply:
Did that in updated commit